### PR TITLE
Updates for the new Injector API

### PR DESCRIPTION
**Pull request by agubler**
**Type:** feature

The following has been addressed in the PR:

* [ ] There is a related issue
* [x] All code has been formatted with [`prettier`](https://prettier.io/) as per the [readme code style guidelines](./../#code-style)
* [x] Unit or Functional tests are included in the PR

**Description:**

Updates to use the latest injector API from `@dojo/widget-core`

### CODE REVIEW

**Review by dylans — APPROVED** [review]

**Review by tomdye — APPROVED** [review]

File: src/StoreInjector.ts [hunk 1]
```diff
@@ -41,60 +40,40 @@ export function storeInject<S>(config: StoreInjectConfig<S>) {
 
 	return handleDecorator((target, propertyKey) => {
 		beforeProperties(function(this: WidgetBase & { own: Function }, properties: any) {
-			const injector: StoreInjector | null = this.registry.getInjector(name);
-			if (injector) {
+			const injectorItem = this.registry.getInjector<Store<S>>(name);
+			if (injectorItem) {
+				const { injector } = injectorItem;
+				const storeInjector = injector();
```
Comment (maier49) [inline src/StoreInjector.ts]: Is this still a storeInjector or just a store at this point?

**Review by maier49 — COMMENTED** [review]
Looks good to me. Had one question

File: src/StoreInjector.ts (hunk 1 above)
Comment (agubler) [reply src/StoreInjector.ts]: done!

**Comment by agubler** [comment]
@maier49 updated the variable name.

**Pull request merged**